test/test.js

```diff
@@ -10,6 +10,12 @@ function sortLeftToRight(a, b) {
 	return 0;
 }
 
+function assertTree(tree, nbElements) {
+	assert.strictEqual(tree._getCount(tree.root), nbElements);
+	assert.strictEqual(tree._isSorted(tree.root), true);
```

Comment:
If you want to test correctness of this code, you can use a method that do not rely on the implementation. You can test for the existence of the element after the addition.

They failed while I was making valid changes. That's why I deleted them.